rtl_433/run.sh

```diff
@@ -1,7 +1,62 @@
 #!/usr/bin/env bashio
 
-CONF_FILE=$(bashio::config "rtl_433_conf_file")
-CONF_FILE="/config/$CONF_FILE"
+conf_directory="/config/rtl_433"
 
-echo "Starting rtl_433 -c $CONF_FILE"
-rtl_433 -c "$CONF_FILE"
+if bashio::services.available "mqtt"; then
+    host=$(bashio::services "mqtt" "host")
+    password=$(bashio::services "mqtt" "password")
+    port=$(bashio::services "mqtt" "port")
+    username=$(bashio::services "mqtt" "username")
+    retain=$(bashio::config "retain")
+else
+    bashio::log.info "The mqtt addon is not available."
+    bashio::log.info "Manually update the output line in the configuration file with mqtt connection settings, and restart the addon."
+fi
+
+if [ ! -d $conf_directory ]
+then
+    mkdir -p $conf_directory
+fi
+
+# Create a reasonable default configuration in /config/rtl_433.
+if [ ! "$(ls -A $conf_directory)" ]
+then
+    cat > $conf_directory/rtl_433.conf.template <<EOD
```

Comment:
"config migration": we could do something like this but with existing config file?